### ISSUE TITLE
Sudden increase in error (validation loss sudden increase )

### ISSUE THREAD

**Issue by alirezashojaei** [jumon/whisper-finetuning#22]
First of all, thanks for your amazing project. 
I'm trying to fine-tune a pretrained model on Persian dataset near 60 thousand audio files. but after some steps, my model get corrupted. the validation loss suddenly increases too much. 
When I run transcription with the model on my test dataset, the characters are totally different from Farsi alphabet.

![Train-1402-09-03](https://github.com/jumon/whisper-finetuning/assets/29631109/9e5805eb-44d8-4e52-81ec-c7d579e1a93f)

This is my transcription result from the last model:

![test-1402-9-4](https://github.com/jumon/whisper-finetuning/assets/29631109/33c78b18-7d02-442b-b457-d29ffd46d591)
